client/index.html

```diff
@@ -83,185 +84,186 @@ <h1 class="title">The Lounge is loading…</h1>
 					<form class="container" method="post" action="">
 						<div class="row">
 							<div class="col-xs-12">
-								<h1 class="title">Sign in to The Lounge</h1>
+								<h1 class="title" data-translate="true">index.signin.title</h1>
 							</div>
 							<div class="col-xs-12">
 								<label>
-								Username
+								<span data-translate="true">index.signin.username</span>
 								<input class="input" name="user">
 								</label>
 							</div>
 							<div class="col-xs-12">
 								<label>
-								Password
+								<span data-translate="true">index.signin.password</span>
 								<input class="input" type="password" name="password">
 								</label>
 							</div>
 							<div class="col-xs-12 error" style="display: none;">
```

Comment:
Are you missing a `data-translate="true"` here?

Reply:
Nice catch